abap-code-review/README.md

```diff
@@ -0,0 +1,25 @@
+# ABAP Code Review Guide
+ABAP Code Review Guide is a collection of ideas, tools, and approaches for implementing ABAP code reviews in various landscapes.
+
+The document is provided as open source under the [CC license](../LICENSE), suggestions and bugfixes are welcome.
+
+Latest release from main can be found out [abap-code-review-guide.pdf](https://github.com/SAP/styleguides/releases/download/latest/abap-code-review-guide.pdf).
```

Comment:
Is the PDF the main reading or just an alternative format?

Reply:
well, both can be read, just the diagrams are only rendered in the PDF